assets/javascripts/modules/addRemove.js

```diff
@@ -7,9 +7,11 @@ A component to add and remove inputs on a page. You have the option of sending i
   `data-can-delete="true"` is set
 - `data-add-btn-text` to send in custom add button text, this defaults to **Add**
 - `data-max` Maximum items allowed in the list
+- each instance's outer div MUST have an ID so that the cloneable item html can be cached for that instance
 
 Markup:
-<div data-add-remove
+<div id="uniqueId"
```

Comment:
I don't think you need an `id` if you recursively search up for `parent(data-add-remove)`